src/core/constraint_template.jl

```diff
@@ -101,7 +121,7 @@ end
 
 
 ""
-function constraint_mc_model_current(pm::AbstractUBFModels; nw::Int=pm.cnw)
+function constraint_mc_model_current(pm::Union{AbstractUBFModels,LPLinUBFPowerModel}; nw::Int=pm.cnw)
```

Comment:
This seems like it should be solved in src/core/types.jl since it seems natural that LPLinUBFPowerModel should be under AbstractUBFModels. If that is not true, the formulation needs to be renamed to avoid confusion

Reply:
I ended up cleaning up the linearizations in #217 , once Frederik reviewed this I will pull it in this branch.